benchmarks/ramspeed/ramspeed_main.c

```diff
@@ -423,7 +429,9 @@ static void memcpy_speed_test(FAR void *dest, FAR const void *src,
 
       if (irq_disable)
         {
+          #ifndef CONFIG_BUILD_KERNEL
```

Comment:
how about define leave_critical_section to empty macro in case of protected/kernel build